source/frontend/src/components/Table/MoneyCell.tsx

```diff
@@ -41,10 +41,10 @@ export const EditableMoneyCell = ({
   );
 };
 
-export const MoneyCell = ({ cell: { value } }: CellProps<IProperty, number | ''>) => (
+export const MoneyCell = ({ cell: { value } }: CellProps<any, number | '' | undefined>) => (
```

Comment:
We use MoneyCell on tables that do not include IProperty data so typescript was complaining. Since I couldn't find a common type I had to use `any`